e2e/bare-https/client.py

```diff
@@ -0,0 +1,35 @@
+import flwr as fl
+import numpy as np
+from pathlib import Path
+
+
```

Comment:
What's the benefit of having objective and model_params here for the outcome of the test? Maybe we can just simplify fit/evaluate? I might be missing something though.

Reply:
the `bare-https` test is copied from `bare` test, which is the simplest E2E test we have. Having a `model_params` can also make sure that flower can aggregate models successfully with HTTPS-enabled. I am open to change if you think we can simplify the fit/evaluate methods here.

Reply:
@tanertopal @panh99 I think it makes sense to keep `bare` and `bare-https` as similar as possible